lib/LANraragi/Plugin/Login/EHentai.pm

```diff
@@ -16,12 +16,13 @@ sub plugin_info {
         type      => "login",
         namespace => "ehlogin",
         author    => "Difegue",
-        version   => "2.2",
+        version   => "2.3",
         description =>
           "Handles login to E-H. If you have an account that can access fjorded content or exhentai, adding the credentials here will make more archives available for parsing.",
         parameters => [
             { type => "int",    desc => "ipb_member_id cookie" },
             { type => "string", desc => "ipb_pass_hash cookie" },
+            { type => "string", desc => "igneous cookie" },
```

Comment:
Could you move `igneous` after `star`? Otherwise I'm expecting this'll break upgrades for users that rely on the star cookie.  

It might also be good to mention that this cookie is optional and required only for proxy scenarios.